### PR TITLE
Fix DB seeds

### PR DESCRIPTION
**Pull request by stepchud**
### Description

Updates the seeds loading script (`priv/repo/seeds_updated.exs`) to correctly load entries in the CSV data. Also removed the duplicate Agencies from `agencies_updated.csv` so that the CSV loads without duplicates, and the number of rows in the CSV matches the number of rows in the DB after seeding.

### Related Issues

A previous PR, #1023 took a different approach to this same problem but it can be closed if this PR is approved/merged.

### Changes Made

- [List the specific changes made in bullet points.]

### Screenshots (if applicable)

[Include screenshots to visually demonstrate changes, if relevant.]

# Checklist
- [ ] PR is assigned to a project: Challenge_gov Board
- [ ] PR is assigned to a milestone: current sprint
- [ ] PR is assigned a reviewer, and requires code review and approval by a teammate
- [ ] New functionality is tested and all tests are green
- [ ] I have added unit tests for new functionality or updated existing tests for changes.
- [ ] I have updated the documentation/README accordingly, if necessary.
- [ ] If you have DB migration, it is a "safe" migration and you've confirmed it can be rolled back.
- [ ] If applicable, Controllers modified contain appropriate authorization plugs
- [ ] This PR has been reviewed by at least one other team member.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -44,7 +44,6 @@ defmodule ChallengeGov.MixProject do
       {:browser, "~> 0.4.4"},
       {:cors_plug, "~> 3.0"},
       {:excoveralls, "~> 0.10", only: :test},
-      {:color_stream, "~> 0.0.1"},
```
Comment (stepchud) [inline mix.exs:47]: this is from #1323

File: mix.lock [hunk 2]
```diff
@@ -6,7 +6,6 @@
   "bunt": {:hex, :bunt, "0.2.1", "e2d4792f7bc0ced7583ab54922808919518d0e57ee162901a16a1b6664ef3b14", [:mix], [], "hexpm", "a330bfb4245239787b15005e66ae6845c9cd524a288f0d141c148b02603777a5"},
   "castore": {:hex, :castore, "0.1.22", "4127549e411bedd012ca3a308dede574f43819fe9394254ca55ab4895abfa1a2", [:mix], [], "hexpm", "c17576df47eb5aa1ee40cc4134316a99f5cad3e215d5c77b8dd3cfef12a22cac"},
   "certifi": {:hex, :certifi, "2.9.0", "6f2a475689dd47f19fb74334859d460a2dc4e3252a3324bd2111b8f0429e7e21", [:rebar3], [], "hexpm", "266da46bdb06d6c6d35fde799bcb28d36d985d424ad7c08b5bb48f5b5cdd4641"},
-  "color_stream": {:hex, :color_stream, "0.0.2", "400fe4b5efe3fcaf723d5bf2ba02f868b92b16b90b0bc99ebcf515a5678412ae", [:mix], [], "hexpm", "b1181f32b310311016006f4f8d52b3418d1af6f06e71903daabafdcaa602a29d"},
```
Comment (stepchud) [inline mix.lock:9]: from #1323

File: mix.lock [hunk 3]
```diff
@@ -81,7 +80,7 @@
...
   "remote_ip": {:hex, :remote_ip, "0.2.1", "cd27cd8ea54ecaaf3532776ff4c5e353b3804e710302e88c01eadeaaf42e7e24", [:mix], [{:combine, "~> 0.10", [hex: :combine, repo: "hexpm", optional: false]}, {:inet_cidr, "~> 1.0", [hex: :inet_cidr, repo: "hexpm", optional: false]}, {:plug, "~> 1.5", [hex: :plug, repo: "hexpm", optional: false]}], "hexpm", "2e7ab1a461cc3cd5719f37e116a08f45c8b8493923063631b164315d6b7ee8e0"},
   "sobelow": {:hex, :sobelow, "0.11.1", "23438964486f8112b41e743bbfd402da3e5b296fdc9eacab29914b79c48916dd", [:mix], [{:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: false]}], "hexpm", "9897363a7eff96f4809304a90aad819e2ad5e5d24db547af502885146746a53c"},
-  "ssl_verify_fun": {:hex, :ssl_verify_fun, "1.1.6", "cf344f5692c82d2cd7554f5ec8fd961548d4fd09e7d22f5b62482e5aeaebd4b0", [:make, :mix, :rebar3], [], "hexpm", "bdb0d2471f453c88ff3908e7686f86f9be327d065cc1ec16fa4540197ea04680"},
+  "ssl_verify_fun": {:hex, :ssl_verify_fun, "1.1.7", "354c321cf377240c7b8716899e182ce4890c5938111a1296add3ec74cf1715df", [:make, :mix, :rebar3], [], "hexpm", "fe4c190e8f37401d30167c8c405eda19469f34577987c76dde613e838bbc67f8"},
```
Comment (stepchud) [inline mix.lock:83]: from #1322

**Review by stepchud — COMMENTED** [review]
let me know if I should remove these lines or wait for those other PRs to be merged

**Review by jdonis — APPROVED** [review]
Remove Duplicates - OK

**Pull request merged**